### PR TITLE
Responsive single blog

### PR DESCRIPTION
**Pull request by Samah-Mustafa**

### CODE REVIEW

File: src/components/common/NavBar.jsx [hunk 1]
```diff
@@ -1,66 +1,95 @@
...
+    if (navItemsPosition === "-100%") {
+      setNavItemsPosition("0");
+    } else {
+      setNavItemsPosition("-100%");
+    }
+  };
+
   return (
-    <>
+    <div className="three">
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: needs better classname

File: src/components/common/NavBar.jsx [hunk 2]
```diff
@@ -1,66 +1,95 @@
 import React from "react";
-import { Nav, Navbar } from "react-bootstrap";
+import { Nav, Navbar, Button, Image } from "react-bootstrap";
 import "./index.css";
 import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [navItemsPosition, setNavItemsPosition] = React.useState("-100%");
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: usually we don't use css in js like this. instead this should be

``[isHamburgerShown, setIsHamburgerShown]```

and it is a boolean

then you add a classname to whatever you need depending on this

**Comment by louisrli** [comment]
hm actually i'm not sure this is ready for review, there's some weird classnames

**Review by louisrli — APPROVED** [review]

File: src/components/common/NavBar.jsx [hunk 3]
```diff
@@ -1,66 +1,95 @@
...
           <Navbar.Brand>
             <img
-              className="brandSpaces"
+              className="suljakLogo"
               src="https://i.ibb.co/L9GcrMY/LLogo-2.png"
               alt="Suljak Logo"
             />
           </Navbar.Brand>
+
+          {/* <Navbar.Brand>
```
Comment (basil-kawak) [inline src/components/common/NavBar.jsx]: delete this commented code

File: src/components/common/NavBar.jsx [hunk 4]
```diff
@@ -1,66 +1,95 @@
...
+          <Nav.Item className="navMargins">
             <Link to="/about" className="navLink">
               About
             </Link>
           </Nav.Item>
 
-          <Nav.Item className="navSpaces">
-            <Link to="/assessment" className="navLink">
+          <Nav.Item className="navMargins">
+            <Link to="/psycho-assess" className="navLink">
```
Comment (basil-kawak) [inline src/components/common/NavBar.jsx]: ```to="/psycho-assess" ``` i think we said to do the paths all lowercase without any other symbols

File: src/components/common/index.css [hunk 5]
```diff
@@ -6,9 +6,23 @@
...
 
 .footer {
-  padding: 5rem 0;
+  padding: 4rem 0;
   background: #014992;
   color: white;
+  height: 500px;
+}
+
+.rr {
```
Comment (basil-kawak) [inline src/components/common/index.css]: change these classes

File: src/components/common/index.css [hunk 6]
```diff
@@ -68,6 +96,9 @@ form button {
   color: #fff;
 }
 
+.socialCircle {
```
Comment (basil-kawak) [inline src/components/common/index.css]: delete this it is empty

File: src/components/common/index.css [hunk 7]
```diff
@@ -82,26 +113,35 @@ form button {
...
   box-shadow: 0 0px 5px rgba(151, 153, 182, 0.034),
     0 5px 13px rgba(127, 124, 161, 0.048),
     0 12.5px 10px rgba(101, 92, 156, 0.06), 0 8px 12px rgba(59, 49, 92, 0.072),
     0 30px 22px rgba(40, 17, 126, 0.086), 0 100px 80px rgba(92, 78, 129, 0.12);
 }
 
-nav.navBar {
-  overflow: hidden;
+nav .navBar {
+  /* overflow: hidden; */
```
Comment (basil-kawak) [inline src/components/common/index.css]: delete comment

File: src/components/common/index.css [hunk 8]
```diff
@@ -121,42 +165,120 @@ nav.navBar {
...
-  width: 60px;
+.humButton {
+  visibility: hidden;
+  position: absolute;
 }
 
-.header {
-  margin-top: 0px;
+@media only screen and (max-width: 900px) {
+  .four {
```
Comment (basil-kawak) [inline src/components/common/index.css]: change class name

File: src/components/common/index.css [hunk 9]
```diff
@@ -121,42 +165,120 @@ nav.navBar {
...
+  .suljakLogo {
+    width: 85%;
+    margin-left: 50%;
+    z-index: 4 !important;
+    visibility: visible;
+  }
+
+  .humButton {
+    visibility: visible;
+    z-index: 80000;
```
Comment (basil-kawak) [inline src/components/common/index.css]: why is it 800000?

File: src/components/common/index.css [hunk 10]
```diff
@@ -121,42 +165,120 @@ nav.navBar {
...
+  }
+
+  .navbarItems {
+    flex-flow: column !important;
+    position: fixed !important;
+    text-align: left !important;
+    padding: 60px 10px 20px;
+    background-color: #2778cc;
+    top: 0;
+    /* left: -100%; */
```
Comment (basil-kawak) [inline src/components/common/index.css]: delete comment

File: src/components/singleBlogPage/index.css [hunk 11]
```diff
@@ -52,11 +70,23 @@
   margin: 1.5rem 0 0.6rem 1.8rem;
 }
 
+.clearFloat {
```
Comment (basil-kawak) [inline src/components/singleBlogPage/index.css:73]: what is ```.clearFloat ```

**Review by basil-kawak — APPROVED** [review]
just change class names and merge

File: src/components/common/NavBar.jsx (hunk 4 above)
Comment (gbilgedal) [reply src/components/common/NavBar.jsx]: yes i just changed it to "/assessment" can you leave it like that samah?

**Comment by gbilgedal** [comment]
please dont merge before changing link to "/assessment" again

**Pull request merged**